coinjoin/coinjoin.go

```diff
@@ -219,6 +219,11 @@ func feeForSerializeSize(relayFeePerKb int64, txSerializeSize int) int64 {
 	return fee
 }
 
+func paysHighFees(relayFeePerKb, fee int64, txSerializeSize int) bool {
+	maxFee := feeForSerializeSize(50*relayFeePerKb, txSerializeSize)
```

Comment:
This value (50x over the standard fee rate) may need tweaking.  This same function in dcrwallet defines high fees as paying over 1000x the standard rate.

We need to be cautious setting this too low, because it may disrupt the mixes at the lowest common denomination used for change mixing, as those transactions do not produce any change outputs.

Reply:
After doing the math I believe this check should be for anything paying 104x or more than the required fee.

The lowest common denomination is 0.00262144, and the required fee to be contributed is 0.00002530.

0.00262144 / 0.00002530 = 103.6

So paying up to 103x can actually occur normally.  But 104x or more indicates a problem.

Reply:
Or 121 because the required fee without the change output is actually lower (0.0000217).

0.00262144 / 0.0000217 = 120.8